eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/NormalStateSpec.scala

```diff
@@ -69,7 +69,7 @@ class NormalStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     val initialState = alice.stateData.asInstanceOf[DATA_NORMAL]
     val sender = TestProbe()
     val h = randomBytes32
-    val add = CMD_ADD_HTLC(MilliSatoshi(50000000), h, CltvExpiryDelta(144).toCltvExpiry, TestConstants.emptyOnionPacket, upstream = Left(UUID.randomUUID()))
+    val add = CMD_ADD_HTLC(MilliSatoshi(50000000), h, CltvExpiryDelta(144).toCltvExpiry(alice.underlyingActor.nodeParams.currentBlockHeight), TestConstants.emptyOnionPacket, upstream = Left(UUID.randomUUID()))
```

Comment:
It might be a bit clearer if the StateTestsHelperMethods fixture exposed `currentBlockHeight` directly (or maybe `aliceCurrentBlockHeight` and `bobCurrentBlockHeight` to be able to test when their block count is desynchronized).

Reply:
I did that at first, then removed it with 2154d98d4d6f3518030ae3f890acab65636a327a because it looked much cleaner. We could still add a `currentBlockHeight` arg with a default value when we need it.